### PR TITLE
Ieee

### PR DESCRIPTION
**Pull request by opoudjis**
### Metanorma PR checklist

 - [ ] Breaking changes (list related PRs)
 - [ ] Documentation update required ([create task for this](https://github.com/metanorma/metanorma.org/issues/new))
 - [ ] External dependency introduced ([documentation update need](https://github.com/metanorma/metanorma.org/issues/new))
 - [ ] Gem with native library introduced

### CODE REVIEW

File: author/ieee/ref/document-attributes.adoc [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+`wg_vicechair`:: The working group vice-chair
+`wg_secretary`:: The working group secretary
+`wg_members`:: The working group members, semicolon-delimited
+
+`balloting_group_members`:: The balloting group members, semicolon-delimited
+`std_board_chair`:: The standards board chair
+`std_board_vicechair`:: The standards board vice-chair
+`std_board_pastchair`:: The standards board past chair
+`std_board_secretary`:: The standards board secretary
+`std_board_members`:: The standards board members, semicolon-delimited. Any emeritus members should be given with an asterisk at the end of their names
```
Comment (ronaldtse) [inline author/ieee/ref/document-attributes.adoc:77]: @opoudjis we have long used the agreed conventions that all document attributes use the "hyphen", not "underscores".

File: author/ieee/ref/document-attributes.adoc [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+`:doctitle:`:: Document title (also can be given as the first line of the Asciidoctor markup, prefaced by `= `.
+Do not include the words "Draft (Guide/Stadnard/Recommended Practice) for", they are supplied automatically
+by Metanorma.
+
+== Document participants
+
+`society`:: The IEEE society responsible for the document. Do not include the "IEEE" prefix.
+`committee`:: The committee responsible for the document.
+`working-group`:: The working group responsible for the document.
+`balloting-group`:: The Standards Association balloting group responsible for the document.
```
Comment (ronaldtse) [inline author/ieee/ref/document-attributes.adoc:59]: Missing the "Pnnn" working group identifier, e.g. "P2874".

File: author/ieee/topics/markup.adoc [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+=== Annexes
+
+Normative and informative annexes are differentiated with the `normative` attribute:
+
+[source,asciidoc]
+----
+[appendix,normative=true]
+== Rules for implementation
+
+[appendix,normative=false]
```
Comment (ronaldtse) [inline author/ieee/topics/markup.adoc:93]: Is this normative syntax new? I thought we always used `obligation=normative`?

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: author/ieee/ref/document-attributes.adoc (hunk 1 above)
Comment (opoudjis) [reply author/ieee/ref/document-attributes.adoc:80]: Done

File: author/ieee/topics/markup.adoc (hunk 3 above)
Comment (opoudjis) [reply author/ieee/topics/markup.adoc]: Fixed

**Pull request merged**